app/controllers/admin/api/account/proxy_configs_controller.rb

```diff
@@ -33,15 +35,26 @@ def index
   private
 
   def proxy_configs
-    accessible_services_ids = System::Database.oracle? ? accessible_services.pluck(:id) : accessible_services.select(:id)
-    ProxyConfig.joins(:proxy).where(proxies: { service_id: accessible_services_ids }).by_environment(environment)
+    @proxy_configs ||= ProxyConfig.joins(:proxy)
+      .where(proxies: { service_id: accessible_services.pluck(:id) })
```

Comment:
```suggestion
      .where(proxies: { service_id: accessible_services.select(:id) })
```

Reply:
@guicassolato unfortunately that wouldn't work for all DBs. Anyway, my next PR is to improve the performance of all this and this part will change.